bitnami/cert-manager/templates/controller/service.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.metrics.enabled }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "certmanager.controller.fullname" . }}
```

Comment:
I'd this name since it's intended for metrics:
```suggestion
  name: {{ printf "%s-metrics" (include "certmanager. controller.fullname" .) | trunc 63 | trimSuffix "-" }}
```